app/controllers/api/v2/planning_elements_controller.rb

```diff
@@ -32,6 +32,9 @@ module V2
     class PlanningElementsController < ApplicationController
       helper :timelines, :planning_elements
 
+      # disable CSRF protection since the add-in doesn't support it
+      skip_before_filter :verify_authenticity_token
```

Comment:
Why is that statement only added here and not on the other APIv2 controllers as well?